### PR TITLE
Gobierto Attachments UI

### PR DESCRIPTION
**Pull request by martgnz**
Connects to #722.

### What does this PR do?
This PR implements the Gobierto Attachments UI in Sandbox, as a small Vue app.

### How should this be manually tested?
I'm rendering the partial at `sandbox/admin_users_user.html`.

### CODE REVIEW

File: app/assets/stylesheets/comp-modals.scss [hunk 1]
```diff
@@ -31,4 +31,150 @@
 .mfp-close-btn-in .mfp-close:hover {
 	color: #FFF;
 	border-radius: 0;
-}
\ No newline at end of file
+}
+.modal-attachments {
```
Comment (furilo) [inline app/assets/stylesheets/comp-modals.scss:35]: modal-attachments o something more generic?

Comment (martgnz) [reply app/assets/stylesheets/comp-modals.scss:35]: Maybe modal-slim? The default modal has a different look (with more padding), so I didn't want to break styles used elsewhere.

**Comment by ferblape** [comment]
This can be tested in staging with a few things still pending to discuss with @martgnz:

- [x] when there are no attachments, the height of the element is not enough
- [x] we need to discuss how to fix the markup in the edit attachment form
- [x] style an error message (try to upload the same attachment twice to get an error message)

**Comment by furilo** [comment]
Regarding design: 

**Sidebar**

- [x] "Adjuntos:" Enclose in a `<small>`
- [x] Button: Let's not create a custom style, in any case lets add a generic button style in comp-forms.scss. We can simply use 'button light small'. Same for the button inside the 
- [x] Is a table needed to list the files? It's not tabular data, just a list.
- [x] List items: let's reduce the text size.
- [x] Do we need the padding for the items?
- [x] When editing a file, the modal is not properly displayed. The default `modal mfp-hide` classes are there? How are we building that modal?

**Add file modal**

- [x] Weird column widths. I would not use things like `.modal-attachments table tr td:nth-child(2), .modal-attachments table tr td:nth-child(3)` as you are doing. Let them flow. 
- [x] When dragging and dropping, styles don't change. We would need to show an #000 opacity .5 full screen overlay and some message?
- [x] @martgnz let's not customize the styles for the forms. What's the need?
- [x] After I have dropped a file, the height of the modal is less than the height of the content. Again, why are not we using the default system for modals that we already have in place? It's because of Vue?

**Comment by furilo** [comment]
@ferblape what do you mean re: `we need to discuss how to fix the markup in the edit attachment form`? Do you mean the markup we generate when including a file in the main textarea?

Also: 

- [x] If I click on a file that is in the sidebar, the page reloads itself but nothing happens. Should we send the file to the textarea?

**Comment by ferblape** [comment]
@furilo I meant to fix the edit form.  I guess this is a markup issue because there is no background color:

screen shot 2017-07-07 at 11 43 49

**Comment by furilo** [comment]
This is because of the modal issues. How are we generating them? Does Vue allow to keep using jquery.magnific-popup ?

**Comment by martgnz** [comment]
The modal is being generated as a vanilla div, like [this example](https://vuejs.org/v2/examples/modal.html). I am looking at a Vue + magnific-popup integration, but it's another layer of abstraction.

After looking at that example, wouldn't be better to fix the styles and reuse the ones we have instead of building the modal with magnific popup? (the modal works, the problem is the css).

**Comment by martgnz** [comment]
@ferblape switched to MagnificPopup and the styles should be OK now. Let's review tomorrow.

**Review by ferblape — APPROVED** [review]
This is OK to me

**Comment by furilo** [comment]
More things: 

- [x] I wouldn't make mandatory the title of the file. 
- [x] If an error gets generated and you keep doing things, the error keeps being there all the time. It should go away at some time.

**Comment by furilo** [comment]
OK let's finish (ie merge) this iteration and create other issues for further improvements.

**Comment by ferblape** [comment]
🚀

**Pull request merged**